app/javascript/components/profile/team-tags-table.vue

```diff
@@ -75,13 +74,18 @@
             />
           </td>
           <td>
-            <team-tags
-              :tags="user.tags"
-              :index="index"
-            />
+            <div :ref="`tags${user.id}`">
+              <team-tags
+                :tags="user.tags "
+                :index="index"
+              />
+            </div>
           </td>
-          <td>
-            <div class="profile-team-tags-table__description">
+          <td class="profile-team-table ">
+            <div
+              class="profile-team-tags-table__description"
+              :style="isMounted && { height: `max(${rowHeight(user.id)}px, 56px)`}"
```

Comment:
tengo la duda de porqué hay que esperar a que esté montado?

Reply:
Porque sino al buscar las la key de la fila que necesito en this.$refs me tira undefined

Reply:
ahh! y porqué no le agregas la clase en el mounted?

Reply:
(el método rowHeight busca this.$refs[`tags${userId}`])  para el div que tiene los tags , para después sacar el clientHeight. Si no está montado no puedo obtener el alto porque this.$refs[`tags${userId}`] es undefined

Reply:
Ahí lo cambié al mounted. Gracias por el consejo!!